src/main/java/org/candlepin/subscriptions/tally/collector/DefaultProductUsageCollector.java

```diff
@@ -41,10 +41,15 @@ public void collect(UsageCalculation prodCalc, NormalizedFacts normalizedFacts)
         else if (!normalizedFacts.isVirtual()) {
             prodCalc.addPhysical(cores, sockets, 1);
         }
-        // Any other system is simply added to the overall total
+        // Any other system should be considered virtual
         else {
-            prodCalc.addToTotal(cores, sockets, 1);
+            prodCalc.addHypervisor(cores, sockets, 1);
```

Comment:
Is this correct? Anything that does not have a cloud provider or is not physical contributes to the overall hypervisor counts? A virtual guest should not be considered a hypervisor in the default case, should it?

Reply:
It doesn't actually matter at the moment. Confirmed w/ Dave that the frontend is just using the total for OpenShift. My concern is that if we have a need at some point to display physical and virtual for OpenShift (or another non-RHEL product), then the total doesn't make sense anymore... I've come to think of hypervisor as a misnomer for "virtual" where "virtual" is generally the footprint of hypervisors (assuming those are reported), but can also include virtual things where the hypervisor is unknown... Thoughts?

Reply:
The reason why you are doing this makes sense, however, the concern I have is that we seem to be using the hypervisor 'category' as  a dumping ground for many things. A system that is virtual doesn't mean it is a hypervisor. I don't want the term hypervisor to become what "Product" has become in candlepin.

That being said, if you feel that it isn't really a big deal, I'm Ok to merge as is.

Reply:
I reverted that particular change. I agree with your thoughts about naming, so if we do end up making such a change, let's also change domain and API as needed (and leave it out of the scope of this PR).

Reply:
:+1: